doc/input-output-reference/src/overview/group-surface-construction-elements.tex

```diff
@@ -1201,6 +1203,19 @@ \subsubsection{Inputs}\label{inputs-13-015}
 
 The ratio, when a sample object is stretched, of the contraction or transverse strain (perpendicular to the applied load), to the extension or axial strain (in the direction of the applied load). This value is used only with complex fenestration systems defined through the Construction:ComplexFenestrationState object. The default value for glass is 0.22.
 
+\paragraph{Field: Window Glass Spectral and Incident Angle Transmittance Data Set Table Name}\label{field-window-glass-spectral-and-incident-angle-transmittance-data}
+
+If Optical Data Type = SpectralAndAngle, this is the name of a spectral and angle data set of transmittance defined with a Table:TwoIndependentVariables object.
+
```

Comment:
Explain here that the x variable must be angle and the y variable must be wavelength, and what the format of the table file should be.  Or explain that in the Table:TwoIndependetVariables object and refer to that here.